crates/diesel_models/src/schema_v2.rs

```diff
@@ -890,12 +882,15 @@ diesel::table! {
         shipping_address -> Nullable<Bytea>,
         capture_method -> Nullable<CaptureMethod>,
         authentication_type -> Nullable<AuthenticationType>,
-        amount_to_capture -> Nullable<Int8>,
         prerouting_algorithm -> Nullable<Jsonb>,
         surcharge_amount -> Nullable<Int8>,
         tax_on_surcharge -> Nullable<Int8>,
         #[max_length = 64]
         frm_merchant_decision -> Nullable<Varchar>,
+        #[max_length = 255]
+        statement_descriptor -> Nullable<Varchar>,
+        enable_payment_link -> Nullable<Bool>,
+        apply_mit_exemption -> Nullable<Bool>,
```

Comment:
Add the below fields,
- customer_present
- payment_link_config
- routing_algorithm_id

Reply:
These fields have been added